wdio.conf.js

```diff
@@ -65,7 +66,9 @@ if (e2eProfile.includes('sauceextended')) {
   driver = 'sauce';
   sauceLabsUsername = process.env.SAUCE_LABS_USERNAME;
   saucelabsAccesKey = process.env.SAUCE_LABS_ACCESS_KEY;
-
+  if (e2eProfile.includes('nightly')) {
```

Comment:
why do you need this? Why don't you set the `MAX_INSTANCE` env variable in `.circle/config.yml`'s nightly workflow?